runtime-config-linux.md

```diff
@@ -242,6 +297,18 @@ The following parameters can be specified to setup the block io controller:
 
 #### Huge page limits
 
+`hugepageLimits` represents the `hugetlb` controller which allows to limit the
+HugeTLB usage per control group and enforces the controller limit during page fault.
+For more information, see the [kernel cgroups documentation about HugeTLB](https://www.kernel.org/doc/Documentation/cgroups/hugetlb.txt).
+
+`hugepageLimits` is an array of entries, each having the following structure:
+
+* **`pageSize`** *(string, required)* - hugepage size
+
+* **`limit`** *(uint64, required)* - limit of *hugepagesize* hugetlb usage
```

Comment:
I think we want “… limit in bytes of…” here.  And probably best to just use “HugeTLB” in all your text references (you currently have “`hugetbl`”, “HugeTBL”, and this “hugetbl”).